src/dotnet-interactive-vscode/src/tests/unit/notebook.test.ts

```diff
@@ -297,6 +309,150 @@ Console.WriteLine(1);
         ]);
     });
 
+    it('diagnostics are reported on CommandFailed', async () => {
```

Comment:
non compiling code produces diagnostics same observation as for the .NET part, we want to assert that diagnostics are done on compiler error and warnings

Reply:
I'd like to leave the events in these tests because the events are faked and it doesn't really matter what the contents of `SubmitCode` actually is; what really matters is the `CommandSucceeded`/`CommandFailed` aspect and I wanted to test exactly that.